eventdata/parameter_sources/randomevent.py

```diff
@@ -260,6 +260,8 @@ def __init__(self, params):
             for d in params['delete_fields']:
                 self._delete_fields.append(d.split('.'))
 
+        self.record_raw_event_size = params.get("record_raw_event_size", True)
```

Comment:
If we set this to `True` we will impact all existing benchmarks by having an additional field. Should this default to `False`?